Automattic/newspack-plugin
# feat(ia): new `WizardsAdminTabs` react app + (new) advertising > sponsors

### Note:
This PR is a dependent on code added in https://github.com/Automattic/newspack-plugin/pull/3270, once that PR has been approved the destination on this PR update to `epic/ia`

### All Submissions:


### Changes proposed in this Pull Request:

1. (new) `WizardsAdminTabs` React app to display a wizard style header component with tabs.

| Before script parse      | After script parse      |
| ------------- | ------------- |
| ![Screenshot 2024-08-02 at 21 24 53](https://github.com/user-attachments/assets/68b02d89-291c-4534-9878-f02935217685) | ![Screenshot 2024-08-02 at 21 24 35](https://github.com/user-attachments/assets/4711c131-41a1-44ae-b475-c5844320fb6f) |

   * Localized data for dynamic composition using `title` and `tabs` props.
   * Core style changes to screen options 
   * Added `is_wizard_page` method to `Newspack\Wizards` and admin body class filter for more deliberate wizards css targeting. 
   * Added `Admin_Tabs` trait to facilitate asset enqueueing and markup output.
2. Migrated Sponsors to Advertising:
   * Added Sponsors listing (edit.php) & Sponsors Settings.
   * Updated `Newspack\Advertising_Sponsors`:
      * Move Newspack Sponsor related menu items to Advertising menu.
      * Manage active menu item using `parent_file` and `submenu_file` filters.

* Miscellaneous:
   * CSS: Fix - Tachyons overrides breaking WP core css.
   * PHP: Added - `Newspack/is_plugin_active()` - similar to core `is_plugin_active` however is available much earlier (vs. only after `plugins_loaded` hook)

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets.
2. Navigate to _/wp-admin/edit.php?post_type=newspack_spnsrs_cpt_ and confirm you see the following:
![Screenshot 2024-08-02 at 21 19 41](https://github.com/user-attachments/assets/eb46d47e-2488-4832-94b4-c73eaa48df32)
3. Add/Edit Sponsors and confirm active menu item is Sponsors (This does not apply to taxonomies, let me know if you feel this is necessary):
![Screenshot 2024-08-02 at 21 22 10](https://github.com/user-attachments/assets/6a9f2242-da84-4bd5-816c-e32d7e026094)
4. Navigate to _/wp-admin/edit.php?post_type=newspack_spnsrs_cpt&page=newspack-sponsors-settings-admin_ and confirm you see the following:
![Screenshot 2024-08-02 at 21 21 23](https://github.com/user-attachments/assets/653d6646-1eb6-4d5f-8d7c-abe3c3f0580f)

### Other information:
